lib/publiccloud/ssh_interactive.pm

```diff
@@ -72,25 +98,24 @@ sub ssh_interactive_leave {
 sub select_host_console {
     my (%args) = @_;
     $args{force} //= 0;
-    my $tunneled = get_var('TUNNELED');
+    my $tunneled = is_tunneled();
+    my $tunnels_initialized = $tunneled && get_var('_SSH_TUNNELS_INITIALIZED');
```

Comment:
still why `$tunnels_initialized` ... this var is used only in next line, so ?

Reply:
That is true, this can be removed.

Reply:
Updated and re-scheduled https://duck-norris.qam.suse.de/tests/11045 as VR for this (small) change.